source/computer/operating-system.rst

```diff
@@ -1,2 +1,51 @@
-操作系统
-========
+选择操作系统
+============
+
+个人电脑有三种常见的操作系统：Microsoft Windows、macOS 和 Linux。
+从市场占有率上看，Microsoft Windows 用户数目远远多于 macOS 和 Linux 用户。
+然而，大多数地震学科研工作者都选择 Linux 或 macOS 作为日常研究的主要操作系统。
```

Comment:
```suggestion
然而，大多数地震学科研工作者都选择 macOS 或 Linux 作为日常研究的主要操作系统。
```

Reply:
Why put macos before Linux?

Reply:
To be consistent with the content. You can also put linux before macOS for the whole 101

Reply:
I believe most people use Linux, so Linux first.

Reply:
It is reasonable. please put linux setup before macOS setup.